Platforms/Linux/src/LinuxPlatformMisc.cpp

```diff
@@ -28,6 +28,10 @@
 
 #include <pthread.h>
 
+#ifdef PLATFORM_FREEBSD
+#    include <pthread_np.h>
```

Comment:
Why is this needed?

Reply:
None of the pthread_*_np stuffs is available without this.

Reply:
Where is this stuff is used?

Reply:
On lines 45 and 61.